v21.1/show-create.md

```diff
@@ -238,6 +239,113 @@ To return just the `create_statement` value:
 
 For more information, see [`COMMENT ON`](comment-on.html).
 
+### Show the `CREATE` statements for all tables, views, and sequences in the current database
```

Comment:
I don't think `Show the `CREATE` statements` covers it all. Probably makes more sense to say `Show the statements needed to recreate all ...` or something along those lines.